spec/project_spec.rb

```diff
@@ -1883,6 +1883,24 @@ def callback(_args); end
           source_info: {}
         ).ordered
 
+        expect(project_instance.notification_center).to receive(:send_notifications).once.with(
+          Optimizely::NotificationCenter::NOTIFICATION_TYPES[:DECISION],
+          'feature', 'test_user', {'browser_type' => 'firefox'},
+          feature_enabled: false,
+          feature_key: 'json_single_variable_feature',
+          source: 'rollout',
```

Comment:
can you add a feature_test test. so, that you also show the proper values in source_info?

Reply:
added source info to one of the tests